src/input/input.container.js

```diff
@@ -1,15 +1,15 @@
 import { connect } from 'react-redux'
-import { change, formValueSelector, getFormSubmitErrors } from 'redux-form'
+import { change, formValueSelector, getFormSyncErrors } from 'redux-form'
 import { formInjector } from '../helpers'
 import Component from './input'
 
 const mapStateToProps = (state, { form, name, placeholder, label, withoutLabel }) => {
-  const submitErrors = getFormSubmitErrors(form)(state)
+  const error = getFormSyncErrors(form)(state)
```

Comment:
Can you explain me why this is better here to use `getFormSyncErrors` ?

Reply:
`getFormSubmitErrors` get error after submit only